src/main/webapp/index.html

```diff
@@ -23,96 +23,136 @@
-      <a class="navbar-brand" href="#">Shopping with photos</a>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
-        aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-      <div class="collapse navbar-collapse" id="navbarCollapse">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item active">
-            <a class="nav-link" href="#">Home<span class="sr-only">(current)</span></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Contact us</a>
-          </li>
-        </ul>
-      </div>
-    </nav>
-  </header>
-
   <main role="main">
-
-    <!-- Wrap the rest of the page in another container to center all the content. -->
-
-    <div class="container feature-display">
-      <h1 class="text-center mb-5">Shop with...</h1>
-      <div class="row">
-        <div class="col-lg-4">
-          <i class="fa fa-image"></i>
-          <h2>Product Photo</h2>
-          <p>
-            Choose an image with the product you are searching for, and we'll compile a shopping 
-            list of similar products for you.
-          </p>
-        </div><!-- /.col-lg-4 -->
-        <div class="col-lg-4">
-          <i class="fa fa-list"></i>
-          <h2>Shopping List</h2>
-          <p>
-            The shopping list may be hand-written.
-          </p>
-        </div><!-- /.col-lg-4 -->
-        <div class="col-lg-4">
-          <i class="fa fa-barcode"></i>
-          <h2>Barcode</h2>
-          <p>
-            Upload a barcode
-          </p>
-        </div><!-- /.col-lg-4 -->
-        
-        <div class="col text-center">
-          <p><button class="btn btn-secondary" id="upload-photo-button">Upload photo »</button></p>
+    <center>
+      <div title="Google" id="google-logo" align="left">
+        <div class="logo-subtext">
+          <span>Shopping</span>
         </div>
+      </div>
+
+      <div class="bar">
+        <input class="searchbar" type="text" title="Search">
+        <img class="camera" 
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Google_Camera_Icon.svg/768px-Google_Camera_Icon.svg.png" 
+            title="Search by Camera"
+            data-toggle="modal" data-target="#upload-image-modal"
+            id="upload-photo-button">
+      </div>
 
-        <!-- Uplad Image Modal -->
-        <div class="bg-model">
-          <div class="model-content">
-            <button class="close" onclick="closeFileUploadDialog()"> + </button>
-            <form id="upload-image-form" class="hidden" action='#' onsubmit="return false" 
-                enctype="multipart/form-data">
-              <h1>Upload image</h1>
+      <button class="button mt-4" data-toggle="modal" data-target="#features-info-modal">
+        Learn more about <em>Shopping with Photos</em>
```

Comment:
Yes, I know. I was going to do this in a further PR, so that the front-end is merged and we have it updated in the master branch, but it's okay, I've done this here 👍 😃